armi/reactor/tests/test_components.py

```diff
@@ -676,6 +766,13 @@ def expansionConservationColdHeightDefined(self, mat: str):
 
 
 class TestTriangle(TestShapedComponent):
+    """Test triangle shaped component.
+
+    .. test:: Triangle shaped component
+        :id: T_ARMI_COMP_SHAPES4
+        :tests: R_ARMI_COMP_SHAPES
+    """
+
```

Comment:
Again, delete this.  Or move it down to one-or-more of the tests in this class.

We can't have class-level test crumbs. Not your fault, you didn't know.